scripts/lib/libvirt/fixtures/cloud-node1-uefi.xml

```diff
@@ -5,8 +5,8 @@
   <vcpu>1</vcpu>
   <os>
     <type arch='x86_64' machine='pc-0.14'>hvm</type>
-<loader readonly='yes' type='pflash'>/usr/share/qemu/ovmf-x86_64-ms-code.bin</loader>
-    <nvram>/var/lib/libvirt/cloud-node1_VARS.fd</nvram>
+<loader readonly='yes' type='pflash'>/usr/share/qemu/ovmf-x86_64-code.bin</loader>
```

Comment:
Because our UEFI netboot loader is built on the crowbar admin node and its not signed by default. To have secure boot enabled we would need to sign it with a valid key (like the one from suse, opensuse, microsoft. etc... ) or build a custom vm firmware that accepts a key that we own (a crowbar key that we dont mind exposing?) and that requires a significant amount of time to setup and deploy.